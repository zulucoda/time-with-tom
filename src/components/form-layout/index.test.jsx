import { render } from '@testing-library/react';
import FormLayout from '.';
import React from 'react';

describe('Components - Form Layout - Unit Test', function () {
  test('does it render', () => {
    const { container } = render(<FormLayout />);
    expect(container).toMatchSnapshot();
  });
});
