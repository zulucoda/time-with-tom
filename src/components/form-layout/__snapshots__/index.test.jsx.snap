// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Form Layout - Unit Test does it render 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthSm"
    >
      <div
        class="MuiBox-root MuiBox-root-12"
      >
        <div
          class="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
        />
      </div>
    </div>
  </div>
</div>
`;
